client/webpack.config.js

```diff
@@ -0,0 +1,77 @@
+/* jshint esversion: 6 */
+const webpack = require('webpack');
+const path = require('path');
+
+const devBuild = process.env.NODE_ENV !== 'production';
+const nodeEnv = devBuild ? 'development' : 'production';
+
+const config = {
+  entry: [
+    'es5-shim/es5-shim',
+    'es5-shim/es5-sham',
+    'babel-polyfill',
+    './app/bundles/Course/LessonPlan/startup',
+  ],
+
+  output: {
+    filename: 'webpack-bundle.js',
```

Comment:
...Shall we rename the out file to `coursemology.bundle.js` ?  It seems more conventional in webpack's document.  

And if we want to split the frontend code into chunks, their names could be `[module-name].bundle.js`

What do you think ?